config/rbac/resourcedistribution_editor_role.yaml

```diff
@@ -0,0 +1,22 @@
+# permissions for end users to edit resourcedistributions.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: resourcedistribution-editor-role
+rules:
+- apiGroups:
+  - apps.kruise.io
+  resources:
+  - resourcedistributions
+  verbs:
+  - get
+  - list
+  - patch
```

Comment:
patch and update for resourcedistributions resources are also not neccessary, i think update and patch for the status subresource of resourcedistributions is enough

Reply:
If so, I think the status subresource of resourcedistributions should not be edited by users as well.

Reply:
I only remove the update and patch for resourcedistribution